### PR TITLE
Added automation for pipeline tasks labels

### PR DESCRIPTION
**Pull request by anatolybolshakov**
Added labels automation for pipeline tasks owned by ABTT

### CODE REVIEW

File: issue-rules.yml [hunk 1]
```diff
@@ -55,24 +55,33 @@ rules:
...
+- valueFor: '**Enter Task Name**'
+  contains: 'InstallSSH'
+  addLabels: ['Area: ABTT', 'Task: InstallSSHKey', 'triage']
 # Area: CrossPlatform
 - valueFor: '**Enter Task Name**'
   contains: 'Android'
   addLabels: ['Area: ABTT', 'triage']
+- valueFor: '**Enter Task Name**'
+  contains: 'AndroidSigning'
+  addLabels: ['Task: AndroidSigning']
```
Comment (damccorm) [inline issue-rules.yml:74]: Should the ABTT label get assigned here too?

File: issue-rules.yml [hunk 2]
```diff
@@ -308,6 +323,11 @@ rules:
   contains: 'PublishCodeCoverageResults'
   addLabels: ['Area: CodeCoverage']
 
+# Area: ABTT
```
Comment (damccorm) [inline issue-rules.yml:326]: Can we group all the ABTT tasks into this section?

File: issue-rules.yml (hunk 1 above)
Comment (anatolybolshakov) [reply issue-rules.yml]: Yep thanks 👍  added 'Area: ABTT' label also here

File: issue-rules.yml (hunk 2 above)
Comment (anatolybolshakov) [reply issue-rules.yml:290]: Sure, I moved it into one section

**Pull request merged**